examples/DotNetCore/EventHub/KeyValueEvent.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using Newtonsoft.Json;
+
+namespace SimpleAppConfigEventHub
+{
+    class KeyValueEvent
+    {
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("topic")]
+        public string Topic { get; set; }
+
+        [JsonProperty("subject")]
+        public string Subject { get; set; }
+
+        [JsonProperty("data")]
+        public Data Data { get; set; }
+
+        [JsonProperty("eventType")]
+        public string EventType { get; set; }
+
+        [JsonProperty("eventTime")]
+        public DateTime EventTime { get; set; }
+
+        [JsonProperty("dataVersion")]
+        public int DataVersion { get; set; }
+
+        [JsonProperty("metadataVersion")]
+        public int MetadataVersion { get; set; }
+    }
+
+    public class Data
```

Comment:
Every class should live in a separate file. You may want to give a more meaningful name to this class.